src/encodings/rle.rs

```diff
@@ -767,7 +768,7 @@ mod tests {
 
   #[test]
   fn test_random() {
-    let seed_len = 10;
```

Comment:
Good question, I went with 32 because I thought I needed 32 u8's to seed the PRNG. Again, I'm not very familiar with the library, but here's the error I see with an array of size 10:

```
   Compiling parquet v0.3.0 (/Users/xlange/code/rust/parquet-rs)
error[E0308]: mismatched types
   --> src/encodings/rle.rs:783:45
    |
783 |       let mut gen = rand::StdRng::from_seed(seed);
    |                                             ^^^^ expected an array with a fixed size of 32 elements, found one with 10 elements
    |
    = note: expected type `[u8; 32]`
               found type `[u8; 10]`
```